legacy/src/app/partials/nodes-list.html

```diff
@@ -1357,6 +1358,14 @@ <h4 class="page-header__dropdown-title">Add rack controller</h4>
                 </span>
               </div>
             </td>
+            <td class="p-table__cell" aria-label="Available upgrade">
+              <div
+                data-ng-show="controller.versions.update"
+                title="{$ controller.versions.update.version $}"
+              >
+                {$ controller.versions.update.version $}
```

Comment:
Do these need to be given the `&&` treatment, or do angular templates fail "gracefully"? Either way I think we should play it safe with angular and have a simple function that checks that an upgrade is available.

Reply:
Good idea.

Reply:
Done.